src/relocation.rs

```diff
@@ -282,4 +336,127 @@ mod tests {
         assert_eq!(trailing_zeros(&[1, 0]), 8);
         assert_eq!(trailing_zeros(&[2, 0]), 9);
     }
+
+    const MAX_AGE: u8 = MIN_AGE + 4;
+
+    proptest! {
+        #[test]
+        fn proptest_actions(
+            peers in arbitrary_unique_peers(2..ELDER_SIZE + 1, MIN_AGE..MAX_AGE),
+            signature_trailing_zeros in 0..MAX_AGE,
+            seed in any::<u64>().no_shrink())
+        {
+            proptest_actions_impl(peers, signature_trailing_zeros, seed).unwrap()
+        }
+    }
+
+    fn proptest_actions_impl(
+        peers: Vec<Peer>,
+        signature_trailing_zeros: u8,
+        seed: u64,
+    ) -> Result<()> {
+        let mut rng = SmallRng::seed_from_u64(seed);
+
+        let sk: bls::SecretKey = rng.gen();
+        let pk = sk.public_key();
+
+        // Create `Section` with `peers` as its members and set the `ELDER_SIZE` oldest peers as
```

Comment:
fort he peers with same age, shall there be a second order of using XorName ?
as the input peers is a Vec, will this causing issue ?